project-base/tests/ShopBundle/Functional/Model/Product/ProductOnCurrentDomainFacadeCountDataTest.php

```diff
@@ -88,10 +88,14 @@ public function categoryTestCasesProvider(): array
      */
     public function testSearch(string $searchText, ProductFilterData $filterData, ProductFilterCountData $expectedCountData): void
     {
-        $filterConfig = $this->productFilterConfigFactory->createForSearch($this->domain->getId(), $this->domain->getLocale(), $searchText);
-        $countData = $this->productOnCurrentDomainFacade->getProductFilterCountDataForSearch($searchText, $filterConfig, $filterData);
+        if ($this->domain->getDomainConfigById(1)->getLocale() === 'en') {
+            $filterConfig = $this->productFilterConfigFactory->createForSearch($this->domain->getId(), $this->domain->getLocale(), $searchText);
+            $countData = $this->productOnCurrentDomainFacade->getProductFilterCountDataForSearch($searchText, $filterConfig, $filterData);
 
-        $this->assertEquals($expectedCountData, $this->removeEmptyParameters($countData));
+            $this->assertEquals($expectedCountData, $this->removeEmptyParameters($countData));
+        } else {
+            $this->markTestSkipped('Tests for product searching are run only when the first domain has English locale');
```

Comment:
Please use an early exit here. Commenting why the test is skipped would be nice.